### PR TITLE
pio-org: add page

### PR DESCRIPTION
**Pull request by marchersimon**
Part of #5517

### CODE REVIEW

File: pages/common/pio-org.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Add a user to an organization:
+
+`pio org add {{orgname}} {{username}}`
+
+- Remove a user from an organization:
+
+`pio org remove {{orgname}} {{username}}`
+
+- List all organizations and their owners, the user owns:
```
Comment (bl-ue) [inline pages/common/pio-org.md:22]: This command does just what I'm saying in the suggestion, right? That's what the [docs](https://docs.platformio.org/en/latest/core/userguide/org/cmd_list.html#description) say.
```suggestion
- List all organizations and their owners:
```

File: pages/common/pio-org.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`pio org list`
+
+- Update the name, email and display name of an organization:
```
Comment (bl-ue) [inline pages/common/pio-org.md:26]:
```suggestion
- Update the name, email or display name of an organization:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
I've left some comments.  Also @marchersimon: I forgot to discuss this in the other PRs (now merged, so we'll need to open a new PR to correct those), but will you please change the variable names to the longer formats in this and your other open PIO PRs, i.e.

- `orgname` → `organization_name`
- `new_orgname` → `new_organization_name`
- `new_displayname` → `new_display_name`
- etc.

File: pages/common/pio-org.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/pio-org.md:22]: Here too, there are far more organizations, than the user owns and this suggestion would imply that it lists all there are.

**Comment by marchersimon** [comment]
@bl-ue those terms don't occur in the already merged PRs, so there's no need for a new one.

File: pages/common/pio-org.md (hunk 1 above)
Comment (bl-ue) [reply pages/common/pio-org.md:22]: It doesn't do that?  The docs appear to me to say that.

Comment (marchersimon) [reply pages/common/pio-org.md:22]: There are many mistakes and bad descriptions in the docs. But at least when I run `pio org list`, I get `You do not have any orginazations`

Comment (bl-ue) [reply pages/common/pio-org.md:22]: How about this?
```suggestion
- List all organizations the current user is a member of and their owners:
```

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**